tests/unit/tests/PaymentGateways/Gateways/Stripe/CreditCardGatewayTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+use Give\Framework\PaymentGateways\Commands\PaymentProcessing;
+use Give\Framework\PaymentGateways\Commands\RedirectOffsite;
+use Give\PaymentGateways\DataTransferObjects\GatewayPaymentData;
+use Give\PaymentGateways\Gateways\Stripe\CreditCardGateway;
+
+/**
+ * @unreleased
+ */
+class CreditCardGatewayTest extends Give_Unit_Test_Case
+{
+    /** @test */
+    public function it_creates_a_payment_that_is_processing()
+    {
+        $_POST['give_stripe_payment_method'] = 'pm_1234';
```

Comment:
@kjohnson I was wondering about using global constants ( $_POST, $_GET) in tests but I find out that the `WP_UnitTestCase` class reset global constants after each test.

👍